content/docs/concepts/traffic-management/virtual-services/index.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Virtual Services
+description: Describes the architecture and behavior of virtual services and provides configuration examples.
+weight: 3
+keywords: [routing, request, traffic routing, requests routing, virtual service, subset, routing rules]
+aliases:
+---
+
```

Comment:
As per comment above, I think we should combine this and the routing rules doc.

Reply:
Also I have a few more edits to do on this doc but I need to catch a train and I'm aware this is getting a bit epic! :)